services/sn_analysis_service/app.py

```diff
@@ -71,7 +92,7 @@ async def run_sn_model(dd):
     # the following code transforms these inputs from SkyPortal
     # to the format expected by sncosmo.
     #
-    results = {"status": "success", "message": "", "data": {}}
+    rez = {"status": "success", "message": "", "analysis": {}}
```

Comment:
Safer to start with failure, perhaps, and only update on success.

Reply:
I'm a half-full kind of guy but ok.